pkg/portworx/pxAuthops.go

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
name of the file should not be in camelCase. instead use px-auth-ops or px-authops.